### PR TITLE
add org membership to library responses (permissions checking)

### PR DESCRIPTION
**Pull request by camhashemi**
@andrewconner  @gratimax this was requested by @DerekBurch to make permissions checking simpler for clients

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryInfoCommander.scala [hunk 1]
```diff
@@ -704,9 +706,11 @@ class LibraryInfoCommanderImpl @Inject() (
       } else {
         (0, 0, Seq.empty)
       }
-      val orgInfoOpt = lib.organizationId.map(basicOrgById.apply)
+      val orgViewOpt = lib.organizationId.map(orgViewById.apply)
```
Comment (maxov) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryInfoCommander.scala:709]: doesn't seem like this is used

File: kifi-backend/modules/shoebox/test/com/keepit/controllers/website/LibraryControllerTest.scala [hunk 2]
```diff
@@ -700,55 +700,12 @@ class LibraryControllerTest extends Specification with ShoeboxTestInjector {
...
-                "permissions":${Json.toJson(lib1.permissionsByAccess(LibraryAccess.OWNER) ++ permissionsFromOrg)}
-               },
-               "invite": null
-              },
-             "subscriptions": [],
-             "suggestedSearches": {"terms": [], "weights": []}
-            }
-          """)
-        Json.parse(contentAsString(result1)) must equalTo(expected)
+        val resultJson = contentAsJson(result1)
```
Comment (maxov) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/website/LibraryControllerTest.scala:705]: :heart:

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryInfoCommander.scala (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryInfoCommander.scala:709]: `orgViewOpt`? It's used in the line below

Comment (maxov) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryInfoCommander.scala:709]: oh, whoops, nvm

**Pull request merged**